Makefile

```diff
@@ -1,30 +1,26 @@
-CASK ?= cask
+EASK ?= eask
 EMACS ?= emacs
 
-cask_dir_exists := $(shell find ./ -type d -name ".cask")
-
 all: test
 
-test: clean-elc
-	$(if $(cask_dir_exists), $(skip), ${MAKE} install-dependencies)
-	${MAKE} unit
+# TODO: Add `test` when it's stable!
```

Comment:
Can you expand on what you mean by stable here? The tests in the repo fail currently?

Reply:
Yes, the tests are failing. I don't have the knowledge regarding the tests in this repo, so I have temporarily disabled it. I recommend opening another PR to fix the tests. 🤔 

See log in https://github.com/jcs-PR/nim-mode/actions/runs/6343959389/job/17233002351.

Reply:
Ok, fair enough! Once I have more time I'll try to take a look at them.